src/scripts/main.js

```diff
@@ -1,3 +1,40 @@
 'use strict';
 
 // write code here
+const getList = document.querySelectorAll('li');
+const arr = Array.from(getList);
+
+function sortList(list) {
+  const sorted = list.sort((a, b) => {
+    const salaryA = parseFloat(
+      a.getAttribute('data-salary').replace(/[^0-9.-]+/g, ''),
+    );
+    const salaryB = parseFloat(
+      b.getAttribute('data-salary').replace(/[^0-9.-]+/g, ''),
+    );
+
+    return salaryB - salaryA;
+  });
+  const removeUl = document.querySelector('ul');
```

Comment:
Ensure that the `ul` element exists before attempting to manipulate it. This will prevent potential runtime errors if the `ul` element is not present in the DOM.